resources/views/profile/partials/update-profile-information-form.blade.php

```diff
@@ -0,0 +1,88 @@
+<section>
+    <header>
+        <h2 class="text-lg font-medium text-gray-900 ">
+            {{ __('Profile Information') }}
+        </h2>
+
+        <p class="mt-1 text-sm text-gray-600 dark:text-gray-400">
+            {{ __("Update your account's profile information and email address.") }}
+        </p>
+    </header>
+
+    <form id="send-verification" method="post" action="{{ route('verification.send') }}">
+        @csrf
+    </form>
+
+    <form method="post" action="{{ route('profile.update') }}" class="mt-6 space-y-6">
+        @csrf
+        @method('patch')
+
+        <div>
+            <x-label for="first-name" :value="__('First name')" />
+            <x-input id="first-name" name="first_name" type="text" class="block w-full mt-1" :value="old('first_name', $user->first_name)" required autofocus autocomplete="first_name" />
+            <x-input-error class="mt-2" :messages="$errors->get('first_name')" />
+        </div>
+
+        <div>
+            <x-label for="last-name" :value="__('Last name')" />
+            <x-input id="last-name" name="last_name" type="text" class="block w-full mt-1" :value="old('last_name', $user->last_name)" required autofocus autocomplete="last_name" />
+            <x-input-error class="mt-2" :messages="$errors->get('last_name')" />
+        </div>
+
+        <div>
+            <x-label for="username" :value="__('Username')" />
+            <x-input id="username" name="username" type="text" class="block w-full mt-1" :value="old('username', $user->username)" required autofocus autocomplete="username" />
+            <x-input-error class="mt-2" :messages="$errors->get('username')" />
+        </div>
+
+        <div>
+            <x-label for="address" :value="__('Address')" />
+            <x-input id="address" name="address" type="text" class="block w-full mt-1" :value="old('address', $user->address)" required autofocus autocomplete="address" />
+            <x-input-error class="mt-2" :messages="$errors->get('address')" />
+        </div>
+
+        <div>
+            <x-label for="phone" :value="__('Phone')" />
+            <x-input id="phone" name="phone" type="tel" class="block w-full mt-1" :value="old('phone', $user->phone)" pattern="[0-9]{3}[0-9]{3}[0-9]{4}" maxlength="10" required autofocus autocomplete="phone" />
+            <x-input-error class="mt-2" :messages="$errors->get('phone')" />
+        </div>
+
+        <div>
+            <x-label for="email" :value="__('Email')" />
+            <x-input id="email" name="email" type="email" class="block w-full mt-1" :value="old('email', $user->email)" required autocomplete="email" />
+            <x-input-error class="mt-2" :messages="$errors->get('email')" />
+
+            @if ($user instanceof \Illuminate\Contracts\Auth\MustVerifyEmail && ! $user->hasVerifiedEmail())
```

Comment:
sao phải check instaceof ở FE nhỉ? nó là instanceof hay ko thì bn setup sẵn ở BE rồi mà?